static/app/components/events/viewHierarchy/node.tsx

```diff
@@ -0,0 +1,93 @@
+import styled from '@emotion/styled';
```

Comment:
This was all taken from the deleted `tree.tsx` file. Since we're using the hook instead I only needed the node.